src/components/PatientProfilePage/PatientProfilePage.tsx

```diff
@@ -1,24 +1,33 @@
 import React, { useState, useEffect } from 'react';
-import { useHistory } from 'react-router-dom';
-import '../styles/EventCreationPage.css';
+import { useHistory, NavLink } from 'react-router-dom';
+import '../../styles/EventCreationPage.css';
```

Comment:
is this still necessary? we should be looking to remove references to old css files as we want to move towards styling being defined within components